lib/inspec/rspec_json_formatter.rb

```diff
@@ -266,13 +302,26 @@ def close(_notification)
       output.puts('')
     end
 
-    res = @output_hash[:summary]
-    passed = res[:example_count] - res[:failure_count] - res[:skip_count]
+    controls_res = controls_summary
+    tests_res = tests_summary
+
+    s = format('Profile Summary: %s%d successful%s, %s%d failures%s, %s%d skipped%s',
+               COLORS['passed'], controls_res['passed'], COLORS['reset'],
+               COLORS['failed'], controls_res['failed']['total'], COLORS['reset'],
+               COLORS['skipped'], controls_res['skipped'], COLORS['reset'])
+    output.puts(s) if controls_res['total'] > 0
+
+    s = format('Test Summary: %s%d successful%s, %s%d failures%s, %s%d skipped%s',
+               COLORS['passed'], tests_res['passed'], COLORS['reset'],
+               COLORS['failed'], tests_res['failed'], COLORS['reset'],
+               COLORS['skipped'], tests_res['skipped'], COLORS['reset'])
+    output.puts(s) if !@anonymous_tests.empty?
+
     s = format('Summary: %s%d successful%s, %s%d failures%s, %s%d skipped%s',
-               COLORS['passed'], passed, COLORS['reset'],
-               COLORS['failed'], res[:failure_count], COLORS['reset'],
-               COLORS['skipped'], res[:skip_count], COLORS['reset'])
-    output.puts(s)
+               COLORS['passed'], 0, COLORS['reset'],
+               COLORS['failed'], 0, COLORS['reset'],
+               COLORS['skipped'], 0, COLORS['reset'])
+    output.puts(s) if @current_control.nil? || (@anonymous_tests.empty? && controls_res['total'] == 0)
```

Comment:
Please remove this old-school summary and use the Test summary instead (adjusting functional tests)